universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java

```diff
@@ -135,29 +135,34 @@ public ProgramDefinition getProgramDefinition(long id) throws ProgramNotFoundExc
 
   @Override
   @Transactional
-  public ProgramDefinition addBlockToProgram(long programId) throws ProgramNotFoundException {
+  public ErrorAnd<ProgramDefinition, CiviFormError> addBlockToProgram(long programId)
+      throws ProgramNotFoundException {
     ProgramDefinition programDefinition = getProgramDefinition(programId);
     String blockName = String.format("Block %d", getNextBlockId(programDefinition));
 
-    return addBlockToProgram(programId, blockName, "", ImmutableList.of());
+    return addBlockToProgram(programId, blockName, "no description", ImmutableList.of());
```

Comment:
suggestion: maybe "default description"? It seems odd to me that we are writing blocks with placeholder descriptions

Reply:
Maybe we can give some instructions to the admin, like "What is the purpose of this block? Add a description that summarizes the information collected"

That way they have a call to action

Reply:
Can do. Done.